Nos.xcodeproj/project.pbxproj

```diff
@@ -2814,7 +2814,7 @@
 			repositoryURL = "https://github.com/pointfreeco/swiftui-navigation";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 1.0.0;
+				minimumVersion = 0.6.1;
```

Comment:
I'm curious whether this is the most recent version that doesn't depend on SwiftSyntax. Not sure if we get any benefits from upgrading but maybe it makes sense to be on the latest non-slow version of `swiftui-navigation`.